charts/nextcloud/Chart.yaml

```diff
@@ -23,7 +23,7 @@ maintainers:
   email: jeff@billimek.com
 dependencies:
 - name: postgresql
-  version: 9.8.9
+  version: 10.3.7
```

Comment:
I think we should handle the postgresql version bump on another PR unless the version bump is required/related to the securityContext

Reply:
I understand, we forked the nextcloud helm chart on our side to handle securityContext and made some updates on postgres. 

securityContext was already handle in the version defined on 9.8.9.

I'll rollback that part